### PR TITLE
Clean local sites

### PR DESCRIPTION
**Pull request by eteq**
This introduces a change to the site-getting machinery so that the *only* local site is the Greenwich Royal Observatory.

This springs from some discussions in astropy-data about how to keep the online and built-in version in sync.  It seemed like it was going to be rather confusing and a headache, particularly with regards to what observatories "make it" into the local copy.  So this PR removes *everything* in the local copy instead of an example case (greenwich).  This is not ruling out the possibility that *future* versions will incorporate observatories into the built-in file, but for now I think it's better to work that all out on the astropy-data version and only later, when it's a bit more stable, start bring things into the built-in file.

cc @astrofrog

### CODE REVIEW

**Comment by eteq** [comment]
Note that one of the remote-data tests will fail as of right now because the greenwich entry is not yet in the online file.  But I can fix that once this is otherwise approved.

**Comment by astrofrog** [comment]
This looks good to me :+1: and my PR (#4320) adds documentation explaining how to get the coordinates of other observatories to work offline.

**Comment by bsipocz** [comment]
:+1: This is way much better, thanks.

File: astropy/coordinates/tests/test_sites.py [hunk 1]
```diff
@@ -56,17 +48,17 @@ def test_online_stes():
...
-    lon, lat, el = keckel.to_geodetic()
-    assert_quantity_allclose(lon, -1*Longitude('155:28.7', unit=u.deg),
-                             atol=0.001*u.deg)
-    assert_quantity_allclose(lat, Latitude('19:49.7', unit=u.deg),
+    greenwichel = EarthLocation.of_site('greenwich')
+    lon, lat, el = greenwichel.to_geodetic()
+    assert_quantity_allclose(lon, -1*Longitude('51:28:30', unit=u.deg),
                              atol=0.001*u.deg)
-    assert_quantity_allclose(el, 4160*u.m, atol=1*u.m)
+    assert_quantity_allclose(lat, Latitude('0:0:0', unit=u.deg),
```
Comment (bsipocz) [inline astropy/coordinates/tests/test_sites.py]: Isn't longitude and latitude the other way around?

Comment (eteq) [inline astropy/coordinates/tests/test_sites.py]: Oh, duh, good point.  Will fix that (it was skipped because it's remote_data...)

**Comment by eteq** [comment]
Thanks @bsipocz and @astrofrog.  I'll push greenwich up to astropy-data and then do the fix @bsipocz suggested and if the remote data then passes this should be good.

**Comment by eteq** [comment]
Also, I can't help but note this is PR 4-3-2-1.  Ready for lift-off! :rocket:

**Comment by eteq** [comment]
OK, this should be all set and I just tested and remote-data also passes.  Once the tests finish this should be good to go.

**Comment by eteq** [comment]
backported to v1.1.x in dd4aa26e0b4370c325c7571214ed1226f1692490

**Pull request merged**